Gruntfile.js

```diff
@@ -291,7 +287,7 @@ module.exports = function( grunt ) {
 	grunt.loadNpmTasks( 'grunt-rtlcss' );
 	grunt.loadNpmTasks( 'grunt-postcss' );
 	grunt.loadNpmTasks( 'grunt-stylelint' );
-	grunt.loadNpmTasks( 'grunt-contrib-jshint' );
+	grunt.loadNpmTasks("gruntify-eslint");
```

Comment:
Coding standards...

Reply:
The joys of copy/paste an an over-reliance on automated fixing tools 😆 

I guess Gruntfile.js isn't checked, I'll fix that too!